lib/Models/SubscriberCustomField.php

```diff
@@ -63,16 +63,17 @@ static function createMultiple($values) {
   }
 
   static function updateMultiple($values) {
+    $subscriber_ids = array_map(function($value) {
```

Comment:
I'd add `array` typehint but actually... you should use `array_column`. I'd also add the `array_unique` from below right here.